crates/control/src/motion/stand_up_back.rs

```diff
@@ -34,6 +36,7 @@ pub struct CycleContext {
 #[derive(Default)]
 pub struct MainOutputs {
     pub stand_up_back_positions: MainOutput<Joints<f32>>,
+    pub stand_up_back_remaining_duration: MainOutput<Duration>,
```

Comment:
I would be in favor of using an `Option<Duration>` here to indicate that there is **some** time left until the robot is upright or that it does not stand up at the moment. Later in your computation you could then just use an `.unwrap_or(Duration::ZERO)`.

Reply:
Good idea, has been changed